lib/matplotlib/figure.py

```diff
@@ -1254,19 +1254,19 @@ def add_subplot(self, *args, **kwargs):
 
         Parameters
         ----------
-        *args, default: (1, 1, 1)
-            Either a 3-digit integer or three separate integers
-            describing the position of the subplot. If the three
-            integers are *nrows*, *ncols*, and *index* in order, the
-            subplot will take the *index* position on a grid with *nrows*
-            rows and *ncols* columns. *index* starts at 1 in the upper left
-            corner and increases to the right.
-
-            *pos* is a three digit integer, where the first digit is the
-            number of rows, the second the number of columns, and the third
-            the index of the subplot. i.e. fig.add_subplot(235) is the same as
-            fig.add_subplot(2, 3, 5). Note that all integers must be less than
-            10 for this form to work.
+        *args, int or (int, int, int) or `SubplotSpec`, default: (1, 1, 1)
+            The position of the subplot described by one of
+
+            - Three integers (*nrows*, *ncols*, *index*). The subplot will
+              take the *index* position on a grid with *nrows* rows and
+              *ncols* columns. *index* starts at 1 in the upper left corner
+              and increases to the right.
+            - A 3-digit integer. The digits are interpreted as if given
+              separately as three single-digit integers, i.e.
+              ``fig.add_subplot(235)`` is the same as
+              ``fig.add_subplot(2, 3, 5)``. Note that this can only be used
+              if there are no more than 9 subplots.
```

Comment:
That's not technically true, you could pass `999` to get the 9th subplot of 81, but I guess we really don't want to encourage this form too much anyway.

Reply:
You're right, in both the observation and the conclusion :smile:.